src/components/tooltip/README.md

```diff
@@ -111,3 +111,4 @@ In the following example, the tooltip text only appears on hover when the button
 | `horizontalConstraint` | `object` |    -     | `xs`, `s`, `m`, `l`, `xl`, `scale`                                                                                                           | `scale` | Horizontal size limit of the tooltip                                                            |
 | `children`             | `node`   |    ✅    | -                                                                                                                                            | -       | Content rendered within the tooltip                                                             |
 | `components`           | `object` |    -     | `WrapperComponent`, `BodyComponent`                                                                                                          | -       | If passed, the tooltip will wrap your component with this element                               |
+| `styles.body`          | `object` |    -     | -                                                                                                                                            | -       | If passed, these styles will be spread onto the div surrounding the tooltip body                |
```

Comment:
the proptypes says it's required.
Should we update the propTypes, or the readme?

Reply:
it's required because it's set with a default though, so for the user it's not required.

Reply:
thanks for the explanation